src/base/bittorrent/torrentinfo.cpp

```diff
@@ -246,6 +246,28 @@ QVector<int> TorrentInfo::fileIndicesForPiece(int pieceIndex) const
     return res;
 }
 
+QByteArray TorrentInfo::hashForPiece(const int pieceIndex) const
+{
+    if (!isValid() || (pieceIndex < 0) || (pieceIndex >= piecesCount()))
+        return {};
+    return { m_nativeInfo->hash_for_piece_ptr(pieceIndex), libtorrent::sha1_hash::size };
+}
+
+QVector<QByteArray> TorrentInfo::piecesHashes() const
```

Comment:
I agree. But still commonly used word referred by me (file names, char codes etc.). IMO, it looks (and sounds) better.